kms/capi/capi.go

```diff
@@ -46,6 +46,17 @@ const (
 	SkipFindCertificateKey = "skip-find-certificate-key" // skips looking up certificate private key when storing a certificate
 )
 
+const (
+	SoftwareProvider  = "Microsoft Software Key Storage Provider"
+	SmartCardProvider = "Microsoft Smart Card Key Storage Provider"
+	PlatformProvider  = "Microsoft Platform Crypto Provider"
```

Comment:
ncrypt_windows.go provides the same constants with a different name, perhaps we want to use those instead of creating new ones. We should also use a constant on New() to set the default provider.